.github/workflows/build-assets.yml

```diff
@@ -54,7 +55,8 @@ jobs:
           echo "community_tag=$community_tag" >> $GITHUB_OUTPUT
           echo "internal_tag=$(echo $community_tag | sed 's/community/internal/')" >> $GITHUB_OUTPUT
           community_base_ref=${{ github.event.base_ref || github.ref }}
-          echo "community_branch=$(echo $community_base_ref | cut -d'/' -f3)" >> $GITHUB_OUTPUT
+          echo "candidate_branch=$(echo $community_base_ref | cut -d'/' -f3)" >> $GITHUB_OUTPUT
+          echo "candidate_base_branch=$(echo $candidate_branch | awk -F'.' -v OFS='.' '{ $3=\"x\"; print }')" >> $GITHUB_OUTPUT
```

Comment:
$candidate_branch will be blank here...  It needs to be set to a local variable before line 58